src/server/src/main.rs

```diff
@@ -52,6 +62,16 @@ async fn hello() -> impl Responder {
     HttpResponse::Ok().body("Hello world!")
 }
 
+#[get("/toggle")]
+async fn toggle(data: web::Data<AppState>) -> impl Responder {
```

Comment:
Any reasons for choosing Actix over Axum?

Reply:
No specific reason. GPT says actix is more feature-rich.